content/usage/advanced/index.md

```diff
@@ -798,6 +798,8 @@ connect to a vehicle:
 {{ easy_image(src="general-config", width=400) }}
 
 - Create and switch the active [User](#users)
+- Manage the application settings directly, including saving, loading, and resetting configurations
```

Comment:
I think a note here mentioning this should be done carefully as it can break Cockpit and stop it from loading is worth it.